src/HTMLGenerator/testSuite/sequencer.py

```diff
@@ -36,8 +37,12 @@ def main():
                     try:
                         with open(json_tests_directory/json_test) as json_test_file:
                             test_cases_json = json.load(json_test_file)
-                            converter_error = convert(top_level_output_directory, test_assets, test_cases_json, Path.cwd().parent)
+                            configuration_file_error = write_configuration_file(test_cases_json, top_level_output_directory, test_assets, Path.cwd().parent)
+                            converter_error = convert(Path.cwd().parent)
 
+                            if configuration_file_error:
+                                test_failures = write_to_dict(test_failures, json_test, configuration_file_error)
```

Comment:
Shouldn't this check come _before_ the converter gets called? If there was an error while creating the config file, we shouldn't be calling the converter, right?

Reply:
Yeah ok